std/encoding.d

```diff
@@ -2462,7 +2462,7 @@ abstract class EncodingScheme
      * Params:
      *    s = the array to be tested
      */
-    bool isValid(const(ubyte)[] s)
+    bool isValid()(const(ubyte)[] s)
```

Comment:
I thought about that, but I was under the impression that that's only true for interfaces and `override` methods don't have to accommodate the attributes?